apps/ldap/utils.py

```diff
@@ -316,3 +320,59 @@ def is_root(username):
 
 def validate_officer(username, password):
     return is_officer(username) and authenticate(username, password)
+
+
+def datetime_to_ldap(dt):
+    """
+    Convert datetime object to LDAP generalized time format
+    """
+    return dt.strftime("%Y%m%d%H%M%S") + "Z"
+
+
+def get_members_older(days=1460):
+    time_threshold = datetime_to_ldap(datetime.now() - timedelta(days=days))
+    with ldap_connection() as c:
+        c.search(
+            PEOPLE_OU,
+            "(createTimestamp<={})".format(time_threshold),
+            attributes="cn"
+        )
+        return [str(entry.cn) for entry in c.entries]
+
+
+def get_members_age_range(older_than=0, younger_than=180):
+    assert younger_than >= older_than and older_than >= 0, "invalid range"
+    young_threshold = datetime_to_ldap(datetime.now() - timedelta(days=older_than))
+    old_threshold = datetime_to_ldap(datetime.now() - timedelta(days=younger_than))
+    with ldap_connection() as c:
+        c.search(
+            PEOPLE_OU,
+            "(&(createTimestamp<={})(createTimestamp>={}))".format(young_threshold, old_threshold),
```

Comment:
```suggestion
            f"(&(createTimestamp<={young_threshold})(createTimestamp>={old_threshold}))",
```
f-strings are more readable